serde/src/ser/impls.rs

```diff
@@ -363,6 +364,32 @@ where
     }
 }
 
+macro_rules! nonzero_integers {
+    ( $( $T: ident, )+ ) => {
+        $(
+            #[cfg(feature = "unstable")]
+            impl Serialize for ::std::num::$T {
+                fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+                where
+                    S: Serializer,
+                {
+                    self.clone().get().serialize(serializer)
```

Comment:
Why is this cloned?

Reply:
I copied this from the generic `NonZero<T>` impl which does not assume `T: Copy` (`get` takes `self` by value), but these concrete types are indeed `Copy` so `Clone` is not necessary. I’ve removed it.